Library/Formula/sassc.rb

```diff
@@ -20,7 +20,7 @@ class Sassc < Formula
   depends_on "libsass"
 
   def install
-    ENV["SASSC_VERSION"] = "3.1.0"
+    ENV["SASSC_VERSION"] = "3.2.0"
```

Comment:
Ok. Maybe it's worth switching this formula to use a Git checkout so this doesn't need to be set/updated each time?